nessie/src/test/java/org/apache/iceberg/nessie/TestNamespace.java

```diff
@@ -73,6 +77,51 @@ public void testListNamespaces() {
     Assertions.assertThat(namespaces).isNotNull().hasSize(2);
   }
 
+  @Test
+  @NessieApiVersions(versions = NessieApiVersion.V1)
+  public void testNamespacesWithV1Api() {
+    createNamespacesAndTables();
+
+    testListTables();
+
+    List<Namespace> namespaces;
+    namespaces = catalog.listNamespaces(Namespace.of("foo"));
+    // NessieApiV1 should include both implicit and explicit namespaces
+    Assertions.assertThat(namespaces)
+        .isNotNull()
```

Comment:
nit: `isNotNull` is also redundant :) Sorry, I missed this in my previous comment.